010_computing_sum_and_product_of_1_to_n.js

```diff
@@ -0,0 +1,29 @@
+const readline = require("readline-sync");
+
+const selectOperation = (choice) => {
+    if ( choice == "m") {
```

Comment:
use ===

Reply:
I have used "==" in order to avoid case  sensitivity in selecting the choice.